stanovy_vpsFree.cz.md

```diff
@@ -176,6 +188,48 @@ nápravu nepřesností ještě před rozesláním.
 výsledků hlasování
   7. obsah námitky člena Spolku týkající se rozhodnutí Schůze členů, jestliže
 o to dotyčný žádá
+6. Umožňuje se konání náhradní Schůze členů dle ustanovení § 257 občanského zákoníku.
+
+### §11a Jednání na dálku (per rollam)
+
+1. Jednání Schůze členů dle § 11 může probíhat způsobem umožňujícím dálkovou
+účast; taková schůze může probíhat částečně či zcela ve virtuálním prostoru
+prostřednictvím prostředků komunikace na dálku, a to v informační systému
+(prostředí) Spolku, event. jiným způsobem (naříklad emailem).
```

Comment:
Doplnil bych `jiným, avšak dostatečně důvěryhodným způsobem`, e-mail za takový nepovažuji, a proto bych jej odstranil z příkladu.

Reply:
Toť otázka. Když tam napíšeš "dostatečně důvěryhodným způsobem", tak si moc nepomůžeš, protože stejně pak někde jinde budeš muset tu "dostatečnou důvěryhodnost" definovat.

Mailem, když se použije nějakej rozumnej algoritmus (Oskar by určitě věděl jakej :) ), to může být klidně taky "dostatečně důvěryhodné". 

Spíš bych tam místo toho příkladu napsal něco jako "jiným způsobem stanoveným  radou nebo vnitřním předpisem".

Reply:
Co emaily podepsané gpg klíčem, který by bylo možné nahrát do vpsfre rozhraní u člena?

Reply:
tohle jsou imho technikálie, který patří když už někam, tak do toho interního předpisu (viz nový bod 1-4. a 6., §18)

Reply:
Myslím si, že není možné nutit členy pořizovat si gpg klíč, vůbec se s ním učit a něco složitě nastavovat. Souhlasím každopádně s tím, že e-mail není věrohodný způsob a vpsAdmin po přihlášení by to řešil elegantně.

Reply:
Commit 54a270b, konkrétní technický řešení do stanov (imho) nepatří.

Jestli máte někdo jinej názor (a návrh konkrétního znění), sem s ním ;) Jinak to zavřu.

Reply:
Souhlasím, je to technikálie, která patří jinam. Takže zvolené řešení je fajn.